pkg/webhooks/resource/validation/validation.go

```diff
@@ -48,6 +64,7 @@ func (v *validationHandler) HandleValidation(
 	admissionRequestTimestamp time.Time,
 ) (bool, string, []string) {
 	if len(policies) == 0 {
+		go v.handleAudit(policyContext.NewResource, request, namespaceLabels)
```

Comment:
can we add a comment here , why we need to audit the reports if there are no policies

Reply:
because we may have audit policies (the policies here only refer to enforced policies).
will add a comment.

Reply:
Added a comment.